packages/mui-material/src/Divider/Divider.js

```diff
@@ -257,6 +257,7 @@ Divider.propTypes /* remove-proptypes */ = {
   /**
    * If `true`, the divider will have a lighter color.
    * @default false
+   * @deprecated Use <Divider sx={{ bgcolor: '#eee' }} /> instead.
```

Comment:
`light` prop was removed from Divider v7 , so deprecated it.

Reply:
```suggestion
   * @deprecated Use <Divider sx={{ bgcolor: '#eee' }} /> (or any color) instead.
```

Reply:
```suggestion
   * @deprecated Use <Divider sx={{ bgcolor: '#eee' }} /> (or any color) instead.
```